core/src/avm2/activation.rs

```diff
@@ -1332,6 +1332,11 @@ impl<'a, 'gc> Activation<'a, 'gc> {
     ) -> Result<FrameControl<'gc>, Error<'gc>> {
         let args = self.pop_stack_args(arg_count);
         let multiname = self.pool_multiname_and_initialize(method, index)?;
+        avm_debug!(
```

Comment:
This feels very duplicate with `avm_debug!(self.avm2(), "Opcode: {op:?}");`.
As in, I understand why you added it - to log the actual names, I also had this need several times. But still, this also makes the log double verbose :/
(and if we're already doing it this way, I'd consider logging them in a slightly more jpexs-like format, at least `Call property`->`callproperty`)

Reply:
I'll drop the commit for now. I originally wanted to replace the opcode debug with a custom one that would replace all the names, that feels more proper - it's just so much work 😅